site/content/docs/5.0/forms/overview.md

```diff
@@ -18,6 +18,8 @@ sections:
     description: Replace browser default range inputs with our custom version.
   - title: Input group
     description: Attach labels and buttons to your inputs for increased semantic value.
+  - title: Floating labels
+    description: Create beautifully simple forms labels that float over your input fields.
```

Comment:
Should "forms" be "form" here and in the new page description?

Reply:
Yes it should be singular